### ISSUE TITLE
Image Export Folder not named after Gpap File

### ISSUE THREAD

**Issue by AndrewM-** [geopaparazzi/geopaparazzi#537]
The number string after the geopaparazzi_images_ is the time stamp when the images were exported.  I thought it was the timestamp of when the gpap file was created and that the gpap file and image export folder would share the same timestamp.  
If I have several gpap files and I export images from one, I can't tell which gpap file the images came from by the folder name.  That creates some issues with file management down the road.  Same goes with the naming convention of kmz files.

**Comment by moovida** [comment]
I agree that this is a good idea. Maybe the project name + some timestamp. The same for kmz and gps. 
Noted
